setup.py

```diff
@@ -25,6 +25,7 @@
 
 with open(os.path.join(here, 'requirements.txt')) as requirements_fp:
   install_requires = requirements_fp.read().split('\n')
+install_requires.append("chevron @ git+https://github.com/taylorjacklespriggs/chevron@support-custom-escaping")
```

Comment:
https://github.com/noahmorrison/chevron/pull/107

Reply:
This isn't a cannonical address (since we can update the branch and it will install the newer updates). Not sure if that's better or not, but wanted to mention